app/models/agents/webhook_agent.rb

```diff
@@ -88,7 +88,7 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [response_message, code]
+      [interpolated(params)['response'] || interpolated['response'] || 'Event Created', code]
```

Comment:
I agree, we don't need it. 

If we want to default to `Event Created` when the interpolated result is an empty string `interpolated(params)['response'].presence || 'Event Created'` would be an option.